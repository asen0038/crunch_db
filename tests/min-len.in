set a 1 0 5 -1
min a
max a
sum a
len a
bye
