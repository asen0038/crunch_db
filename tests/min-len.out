>
ok

>
-1

>
5

>
5

>
4

>
bye
